components/board/numato/board.go

```diff
@@ -360,16 +360,16 @@ type analog struct {
 	pin string
 }
 
-func (ar *analog) Read(ctx context.Context, extra map[string]interface{}) (int, error) {
-	res, err := ar.b.doSendReceive(ctx, fmt.Sprintf("adc read %s", ar.pin))
+func (a *analog) Read(ctx context.Context, extra map[string]interface{}) (int, error) {
```

Comment:
I think the numato code can either not use the analog smoother or implement a `Close` more gracefully. We'll handle this driver in our next bug bash.